src/sentry/workflow_engine/types.py

```diff
@@ -48,3 +48,4 @@ def evaluate_value(value: T, comparison: Any, condition: str) -> DataConditionRe
 
 class DetectorType(StrEnum):
     ERROR = "ErrorDetector"
+    METRIC_ALERT_FIRE = "metric_alert_fire"
```

Comment:
What are these meant to map to? I feel like we should have a consistent format

Reply:
this maps to the grouptype's slug through the detector. Agree though, i'm thinking we should set these as types on the detector base class.

I was going to do that in a follow-up PR (we also need something similar for data_sources)